youtube.css

```diff
@@ -0,0 +1,155 @@
+html {
+  border: 2px solid gray;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
You shouldn't style the `html` tag, instead of that style the `body` tag as the topmost element, by definition the `body` tag is what gets rendered on the browser.